go/cmd/dolt/commands/cnfcmds/conflictsplitter.go

```diff
@@ -73,6 +73,11 @@ func newConflictSplitter(conflictQuerySch sql.Schema, targetSch sql.Schema) (*co
 			}
 		}
 
+		if colName == "" {
```

Comment:
Under what circumstances does this happen?

Reply:
Right now, the only columns that fits this description are `from_root_ish` and `dolt_conflict_id`.

This code was originally written with the expectation that the fields are returned in a very specific order, but this would have required more work on our part and more queries (we would need to figure out the "base", "their", and "our" schemas, then use that info to craft the `select base_pk, our_pk, their_pk... from dolt_conflicts_FOO` query). Instead of doing that, I chose to slightly modify the code that parses this data, so it could accept whatever order of fields Dolt gives us. Getting all the fields in random order surprisingly *almost* worked (thanks Dhruv for writing such good code!), except we would have instances like these where we'd have to filter out the column.

Reply:
Got it. Resolving.